src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -32,57 +32,57 @@ public abstract partial class DelegatingHandler : System.Net.Http.HttpMessageHan
     public partial class FormUrlEncodedContent : System.Net.Http.ByteArrayContent
     {
         public FormUrlEncodedContent(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>> nameValueCollection) : base (default(byte[])) { }
```

Comment:
Technically I think the `KeyValuePair<string, string>` should be `KeyValuePair<string?,string?>`.  From a quick look at the implementation, it appears to explicitly allow that with a string.IsNullOrEmpty check.

Reply:
yes it is, updating